### DISCUSSION TITLE
Framer Motion

### DISCUSSION THREAD

**Discussion by mohux** [vercel/next.js#15865] [Help]
Am trying to use motion framer with nextjs
but the hot reload is not working with it, have to refresh everytime
any suggestions?

**Comment by rafaelalmeidatk** [comment]
If you mean the animation isn't running when you save the file, that's because Fast Refresh doesn't trigger a remount of the components. You can force it by adding `// @refresh reset` anywhere in the file of the component and now the component will reset every time you save the file, running the animation

**Reply by mohux** [reply]
Hi @rafaelalmeidatk, thanks for your replay, 
no its not working
the issue is that as you mentioned I want it to restart fully, since framer motion relies on initial data to begin the animation attributes

adding @refresh is not working sadly, it works if I add it inside a "page" file 
but doesn't work with components at all
